src/Microsoft.Health.Fhir.Shared.Core/Features/Definition/SearchableSearchParameterDefinitionManager.cs

```diff
@@ -0,0 +1,79 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Features.Search;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.ValueSets;
+
+namespace Microsoft.Health.Fhir.Core.Features.Definition
+{
+    /// <summary>
+    /// A SearchParameterDefinitionManager that only returns actively searchable parameters.
+    /// If a non-searchable parameter is requested a <see cref="SearchParameterNotSupportedException"/> will be raised.
+    /// </summary>
```

Comment:
This seems to be specific to GetSearchParameter(string resourceType, string name)

Reply:
Removed the comment